Lib/_pydatetime.py

```diff
@@ -2419,7 +2422,7 @@ def __new__(cls, offset, name=_Omitted):
         if not cls._minoffset <= offset <= cls._maxoffset:
             raise ValueError("offset must be a timedelta "
                              "strictly between -timedelta(hours=24) and "
-                             "timedelta(hours=24).")
+                             f"timedelta(hours=24), not {offset!r}")
```

Comment:
```suggestion
                             f"timedelta(hours=24), but got {offset!r}")
```

Reply:
@erlend-aasland sounds right. I'll change 'but got ' to 'not '. ok?